README.md

```diff
@@ -1 +1,7 @@
 # ceph.io
+
+:warning: **To report an issue with the [ceph.io](https://ceph.io/) website**: file it with the [live website repository], not here.
```

Comment:
the existing website is not stored in git, issues can be reported to Mike Perez <miperez@redhat.com>, the Ceph community manager

Reply:
Thanks! I've replaced the placeholder with the webmaster's contact info.